jax/lib/__init__.py

```diff
@@ -50,6 +50,7 @@ def _check_jaxlib_version():
 except:
   tpu_client = None
 from jaxlib import xla_client
+from jaxlib import xla_extension as xe
```

Comment:
nit: maybe don't alias xla_extension here, to make it more clear where it's used?

Reply:
Deleted this import entirely.